examples/incompressible_flow/3d_vessels/vessels.prm

```diff
@@ -0,0 +1,140 @@
+# Listing of Parameters
+# ---------------------
+
+
+# --------------------------------------------------
+# Simulation Control
+#---------------------------------------------------
+subsection simulation control
+  set method                  = bdf1
+  set time step               = 0.0001
+  set time end                = 10
+  set output name             = vessels
+  set output path             = ./output/
+  set output frequency        = 10
+end
+
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+subsection FEM
+    set velocity order            = 1
+    set pressure order            = 1
+    set tracer order              = 1
+    set qmapping all              = false
+end
+
+#---------------------------------------------------
+# Multiphysics
+#---------------------------------------------------
+subsection multiphysics
+	set tracer = true
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+    set kinematic viscosity            = 1
+    set tracer diffusivity             = 0.001
+end
+
+subsection timer
+    set type = iteration
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+    set type                 = gmsh
+    set file name            = vessels.msh
+    set initial refinement   = 0
+    set simplex              = true
+end
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+  set number                  = 4
```

Comment:
Why number = 4 if you only set 3 three boundary conditions?
You need to have number = number of boundary conditions you actually set.